client/components/upgrades/google-apps/google-apps-dialog/index.jsx

```diff
@@ -51,10 +51,6 @@ class GoogleAppsDialog extends React.Component {
 		}
 	}
 
-	removeValidationErrors() {
```

Comment:
This function isn't used anywhere. I figure it was copied and pasted when this file was created.